src/world-company-remuneration/salary-calculator-service-impl.ts

```diff
@@ -3,6 +3,20 @@ const K = "K";
  * @author Jack Napier <jnapier@therealjoker.com>
  */
 export function salary(a: number, b: number) {
+    if (a === null){
+        if (666 === a) {
+            return "132" + K;
+        } else {
+            if (b > 10) {
+                return "41,8" + K;
+            } else {
+                if (b > 5) {
+                    return "37,4" + K;
+                } else {
+                    return "33" + K;
+                }
+            }}
+    }
```

Comment:
```suggestion
    if (a === null){
            if (b > 10) {
                return "41,8" + K;
            } else {
                if (b > 5) {
                    return "37,4" + K;
                } else {
                    return "33" + K;
                }
            }
    }
```
since a is already null it can not be a number.